CodeiumVS/SettingsPage.cs

```diff
@@ -39,6 +40,21 @@ public string PortalUrl
         }
     }
 
+    [Category("Codeium")]
+    [DisplayName("Extension Base Url")]
+    [Description("URL of the Codeium Extension Base.")]
```

Comment:
A little more detailed description would be helpful.
```suggestion
    [DisplayName("Language Server Download URL")]
    [Description(
        "If you're experiencing network issues with GitHub and can't download the language server, please change this to a GitHub Mirror URL instead. For example: https://gh.api.99988866.xyz/https://github.com/Exafunction/codeium/releases/download")]
```